Source/Core/UICommon/GameFileCache.cpp

```diff
@@ -99,6 +99,9 @@ bool GameFileCache::Update(
   game_paths.reserve(all_game_paths.size());
   for (const std::string& path : all_game_paths)
   {
+    if (aborted && *aborted)
```

Comment:
This should be an `atomic<bool>` if you want to see changes made from another thread.
The compiler is allowed to optimize away the continuous checking of a regular `bool`. 
It doesn't look like you ever pass a `nullptr`? Please pass a reference to `atomic<bool>` in that case.

Reply:
As suggested, I changed the type to `const std::atomic_bool&`.